waspc/examples/todoApp/src/server/actions.ts

```diff
@@ -1,8 +1,9 @@
 import HttpError from '@wasp/core/HttpError.js'
+import { Task } from '@wasp/entities/'
```

Comment:
Maybe we could use `import type` for type-only imports? It could help with some tooling in the future, and it feels nice to make the distinction. https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-5.html#type-modifiers-on-import-names

```ts
import { type Task } from  '@wasp/entities/'
```